### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AlyonaArlova**
HomeTask2-AlyonaA

### CODE REVIEW

File: src/Lesson2/HomeWork2Task2.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package Lesson2;
+
+import java.util.Scanner;
+
+public class HomeWork2Task2 {
+    public static void main(String[] args) {
+        /*2.1) Дано 2 числа, день и месяц рождения. Написать программу, которая определяет знак зодиака человека.*/
+        Scanner in1 = new Scanner(System.in);
```
Comment (KiperUladzislau) [inline src/Lesson2/HomeWork2Task2.java:8]: Invalid output for date `32.02` for example

File: src/Lesson2/HomeWork2Task3a.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import static java.lang.Math.pow;
+
+/*
+3.2) Вычислить: 1+2+4+8+...+256
+ */
+public class HomeWork2Task3a {
+    public static void main(String[] args) {
+        double sum = 0;
+        for (int i = 1; i < 9; i++)
```
Comment (KiperUladzislau) [inline src/Lesson2/HomeWork2Task3a.java:11]: Always use `{ }` for cycles and if statements, even its only one line of code

File: src/Lesson2/HomeWork2Task3a.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            sum = sum + pow(2, i);
```
Comment (KiperUladzislau) [inline src/Lesson2/HomeWork2Task3a.java:12]: Hacky :)  Try to solve it without `Math#pow`

File: src/Lesson2/HomeWork2Task5.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package Lesson2;
+/*
+ Написать программу,
+ которая печатает на консоль прямоугольник из символа "@" длины NxM
+  (задается вручную, т.е. int n = 10; int m = 5, например)
+ */
+public class HomeWork2Task5 {
+    public static void main(String[] args) {
+        for (int a = 0; a < 4; a++) {
```
Comment (KiperUladzislau) [inline src/Lesson2/HomeWork2Task5.java:9]: Nope, you need to use 2 cycles here
You can only change number of lines in your solution, but can not change number of `@` in one line

File: src/Lesson2/HomeWork2Task2.java (hunk 1 above)
Comment (AlyonaArlova) [reply src/Lesson2/HomeWork2Task2.java:8]: image

Comment (AlyonaArlova) [reply src/Lesson2/HomeWork2Task2.java:8]: for me this case has valid validation - 
 if (day >= 32 || day < 1 || monthBirth < 1 || monthBirth >= 12) {
            System.out.println("ERROR! Please, enter valid data");

**Pull request closed without merging**